pkg/build/api/types.go

```diff
@@ -142,6 +146,9 @@ const (
 
 	// BuildPhaseCancelled indicates that a running/pending build was stopped from executing.
 	BuildPhaseCancelled BuildPhase = "Cancelled"
+
+	// BuildPhaseSkipped indicates that a queued build was skipped.
+	BuildPhaseSkipped BuildPhase = "Skipped"
```

Comment:
I guess this raises a question of what the manual action is going to be.  I thought it would be "delete all the queued builds" in which case we don't need a new phase.  But if the manual action will be "skip all the queued builds except the newest one, and run the newest one" then it's debatable.  There's still an argument you could just delete all the builds that weren't run, but Skipped is probably better for audit purposes.

so in conclusion yes i think we want this, but you'll need to be careful because there are a lot of places in code where we look for builds in a "terminated" state and we need to make sure they now treat this as a terminated state.  I think we have an array somewhere of all the "terminal" states that you can just add this to, but it may not be used consistently.  i'd audit the code for anywhere we reference any of the existing terminal state constants.